src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -1450,6 +1450,7 @@ public static function provideTruncate()
             ['foobar...', 'foobar foo', 6, '...', false],
```

Comment:
Sort-of unrelated, but what happens with `['foobar...', 'foobar', 5, '...'],`
ie. if the string `($length + $ellipsisLength) > $stringLength`?
It should return the full string then, possibly?

Reply:
I'm not sure what you mean, please provide a test case if you want.

Reply:
I was not sure whether the length provided to the method is including or excluding the ellipsis. But apparently it is, so the case I talked about can't happen here.

So nvm then 👍